bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -48,7 +48,7 @@ spec:
           set -euo pipefail
 
           # if another process is listening on the cni-server socket, wait until it exits
-          trap 'kill $(jobs -p); exit 0' TERM
+          trap 'kill $(jobs -p); rm -Rf /etc/cni/net.d/80-openshift-network.conf ; exit 0' TERM
```

Comment:
This seems redundant with the preStop hook... does it buy us anything?

(Also, in all three cases, the `R` flag to `rm` is unnecessary.)

Reply:
@danwinship good question; though perhaps there are cases where the termination hook doesn't fire as we expect. Not sure.

Reply:
@dcbw There is a race form when the processes dies to when the file is deleted. It is good to limit the exposure.